Airports/EGFF/SMAA.txt

```diff
@@ -1,60 +1,70 @@
-                            N051.22.19.667 W003.50.59.625 N051.27.20.606 W003.53.09.880
-                            N051.22.33.033 W003.01.47.347 N051.21.37.133 W003.01.50.551
-                            N051.23.09.692 W003.01.50.761 N051.22.33.033 W003.01.47.347
-                            N051.23.33.118 W003.24.19.719 N051.24.00.172 W003.23.59.210
-                            N051.23.33.118 W003.24.19.719 N051.24.40.255 W003.29.49.695
-                            N051.23.34.584 W003.20.19.048 N051.29.12.403 W003.04.11.992
-                            N051.23.57.673 W003.16.45.191 N051.21.42.878 W003.18.33.896
-                            N051.23.58.880 W003.02.00.592 N051.23.09.692 W003.01.50.761
-                            N051.24.00.172 W003.23.59.210 N051.23.34.584 W003.20.19.048
-                            N051.24.00.172 W003.23.59.210 N051.23.33.118 W003.24.19.719
-                            N051.24.39.796 W003.02.15.513 N051.24.53.174 W002.57.36.442
-                            N051.24.39.796 W003.02.15.513 N051.23.58.880 W003.02.00.592
-                            N051.24.40.255 W003.29.49.695 N051.23.33.118 W003.24.19.719
-                            N051.24.40.255 W003.29.49.695 N051.26.02.443 W003.36.38.177
-                            N051.24.53.174 W002.57.36.442 N051.24.39.796 W003.02.15.513
-                            N051.25.46.819 W003.22.30.992 N051.24.00.172 W003.23.59.210
-                            N051.26.02.443 W003.36.38.177 N051.30.28.774 W003.32.58.063
-                            N051.27.20.606 W003.53.09.880 N051.27.46.183 W003.56.50.920
-                            N051.27.20.606 W003.53.09.880 N051.24.40.255 W003.29.49.695
-                            N051.27.20.606 W003.53.09.880 N051.31.00.861 W003.54.45.039
-                            N051.27.46.183 W003.56.50.920 N051.27.20.606 W003.53.09.880
-                            N051.29.12.403 W003.04.11.992 N051.30.38.216 W003.00.11.353
-                            N051.29.12.403 W003.04.11.992 N051.24.39.796 W003.02.15.513
-                            N051.30.28.774 W003.32.58.063 N051.25.47.902 W003.22.31.896
-                            N051.30.38.216 W003.00.11.353 N051.29.12.403 W003.04.11.992
-                            N051.31.00.861 W003.54.45.039 N051.31.24.841 W003.51.30.527
-                            N051.31.24.841 W003.51.30.527 N051.32.09.121 W003.45.38.317
-                            N051.32.09.121 W003.45.38.317 N051.35.06.939 W003.21.55.447
-                            N051.35.06.939 W003.21.55.447 N051.37.59.792 W003.22.15.807
-                            N051.35.06.939 W003.21.55.447 N051.36.54.168 W003.07.32.421
-                            N051.36.54.168 W003.07.32.421 N051.29.12.403 W003.04.11.992
-                            N051.37.59.792 W003.22.15.807 N051.35.06.939 W003.21.55.447
\ No newline at end of file
+EGFF Cardiff SMAA               S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
```

Comment:
@AndyTWF can I just check that the compiler will sort out any spacing inconsistencies?